### ISSUE TITLE
The latest version 2.2.2 fails - 2.2.0 works

### ISSUE THREAD

**Issue by pbastowski** [Kocal/jsdoc-vuejs#124]
With version 2.2.0 everything works fine. With 2.2.2 I get the error message below. My config file is after the error message.

```bash
> jsdoc -c jsdoc.config.json

internal/modules/cjs/loader.js:550
    throw err;
    ^

Error: Cannot find module './config'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Requizzle.requizzle (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/requizzle.js:87:22)
    at infectProxy (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/loader.js:82:28)
    at Module.targetModule.require (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/loader.js:101:11)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/jsdoc-vuejs/index.js:2:16)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at Object.load (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/loader.js:112:15)
    at Requizzle.requizzle (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/requizzle.js:90:24)
    at infectProxy (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/loader.js:82:28)
    at Module.targetModule.require (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/requizzle/lib/loader.js:101:11)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.exports.installPlugins (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/jsdoc/lib/jsdoc/plugins.js:19:18)
    at Object.module.exports.cli.createParser (/Users/paul/dev/kpmg/SOFY2/frontend/node_modules/jsdoc/cli.js:354:17)
```

**config.json**
```js
{
  "plugins": [
    "node_modules/jsdoc-vuejs",
    "plugins/markdown"
  ],
  "source": {
    "include": [
      "./src",
      "./pages",
      "./plugins",
      "./middleware",
      "./store"
    ],
    "includePattern": "\\.(vue|js)$",
    "excludePattern": "(node_modules/|docs)"
  },
  "opts": {
    "template": "node_modules/docdash",
    "encoding": "utf8",
    "destination": "../docs/nuxt-docs",
    "recurse": true,
    "verbose": false
  },
  "docdash": {
    "search": true
  },
  "templates": {
    "cleverLinks": false,
    "monospaceLinks": false
  }
}```

**Comment by silverspectro** [comment]
same problem here.

**Comment by Kocal** [comment]
Oh sorry, I totally forgot about this issue.
I will take a look.

**Comment by Kocal** [comment]
Fixed in #127, v2.2.3 has been released.

**Comment by pbastowski** [comment]
Thanks, it now works.
